packages/chevrotain/src/parse/grammar/gast/gast_resolver_public.ts

```diff
@@ -27,7 +27,7 @@ export function resolveGrammar(options: {
   forEach(options.rules, (rule) => {
     topRulesTable[rule.name] = rule
   })
-  return orgResolveGrammar(topRulesTable, options.errMsgProvider)
+  return orgResolveGrammar(topRulesTable, options.errMsgProvider!)
```

Comment:
if we define a new `actualOptions` variable would TSC understand that `errMsgProvider` us no longer 
optional?

Reply:
yes

Reply:
i did that